configurations/pipeline_configuration.py

```diff
@@ -42,6 +42,16 @@
             )
         )
     ],
+    csv_sources=[
+        CSVSource(
+            "gs://avf-project-datasets/2022/RVI-ELECTIONS/recovered_hormuud_2022_03_12_to_2022_03_18_de_identified.csv",
+            engagement_db_datasets=[
+                # This contains data from 12th March, until 18th March.
+                CSVDatasetConfiguration("rvi_elections_s01e01", start_date=isoparse("2022-03-12T00:00:00+03:00"))
```

Comment:
end_date?

Reply:
The CSV file is already filtered to contain data up to the 18th, but I've added it.